File: src/main/java/de/itsTyrion/flusenbot/util/InputThread.java
package de.itsTyrion.flusenbot.util;

import com.pengrad.telegrambot.TelegramBot;
import com.pengrad.telegrambot.request.DeleteMessage;
import com.pengrad.telegrambot.request.SendMessage;
import lombok.RequiredArgsConstructor;

import java.util.Arrays;
import java.util.Scanner;

@RequiredArgsConstructor
public final class InputThread extends Thread {
    private final TelegramBot bot;

    @Override
    public void run() {
        Scanner sc = new Scanner(System.in);
        while (sc.hasNextLine()) {
            try {
                String input = sc.nextLine().trim();
                var split = input.split(" ");

                //noinspection IfCanBeSwitch
                if (split[0].equals("stop")) {
                    Runtime.getRuntime().exit(0);
                } else if (split[0].equals("send")) {
                    var joined = String.join(" ", Arrays.copyOfRange(split, 2, split.length));
                    bot.execute(new SendMessage(Long.parseLong(split[1]), joined));
                } else if (split[0].equals("del")) {
                    bot.execute(new DeleteMessage(Long.parseLong(split[1]), Integer.parseInt(split[2])));
                } else if (split[0].equals("welcome"))
                    bot.execute(new SendMessage(Long.parseLong(split[1]), "Willkommen, " + split[2] + "^^"));
            } catch (Exception ex) {
                ex.printStackTrace();
            }
        }
        sc.close();
    }
}

File: src/main/java/de/itsTyrion/flusenbot/util/InputThread.kt
package de.itsTyrion.flusenbot.util

import com.pengrad.telegrambot.TelegramBot
import com.pengrad.telegrambot.request.DeleteMessage
import com.pengrad.telegrambot.request.SendMessage
import java.util.*

class InputThread(private val bot: TelegramBot) : Thread() {

    override fun run() {
        val sc = Scanner(System.`in`)

        while (sc.hasNextLine()) {
            try {
                val input = sc.nextLine().trim { it <= ' ' }
                val split = input.split(" ").toTypedArray()
                when (split[0]) {
                    "stop" -> Runtime.getRuntime().exit(0)

                    "send" -> {
                        val joined = split.copyOfRange(2, split.size).joinToString(" ")
                        bot.execute(SendMessage(split[1].toLong(), joined))
                    }
                    "del" -> bot.execute(DeleteMessage(split[1].toLong(), split[2].toInt()))

                    "welcome" -> bot.execute(SendMessage(split[1].toLong(), "Willkommen, " + split[2] + "^^"))
                }
            } catch (ex: Exception) {
                ex.printStackTrace()
            }
        }
        sc.close()
    }
}